contracts/genie-airdrop/src/contract.rs

```diff
@@ -12,7 +12,7 @@ use genie::airdrop::{
 };
 use genie::asset::{build_transfer_asset_msg, query_balance, AssetInfo};
 
-const CONTRACT_NAME: &str = "genie-airdrop";
+const CONTRACT_NAME: &str = "genie-raffle";
```

Comment:
@urieltan this should not be changed

```suggestion
const CONTRACT_NAME: &str = "genie-airdrop";
```